perl.c

```diff
@@ -3384,7 +3383,13 @@ Perl_get_debug_opts(pTHX_ const char **s, bool givehelp)
 
     if (isALPHA(**s)) {
         /* if adding extra options, remember to update DEBUG_MASK */
```

Comment:
Should this be "if adding or removing options"? `DEBUG_MASK` still has 0x2000 set, though it looks unlikely to cause any problems.

Reply:
FWIW, I plan to reuse 0x2000. So there isn't much point in doing anything about it. I can change the comment when I push my PR to replace the '?' with a new letter.